logsmanagement/rrd_api/rrd_api_systemd.c

```diff
@@ -0,0 +1,206 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#include "rrd_api_systemd.h"
+
+const char *dim_sever_str[SYSLOG_SEVER_ARR_SIZE] = {
+    "0:Emergency",
+    "1:Alert",
+    "2:Critical",
+    "3:Error",
+    "4:Warning",
+    "5:Notice",
+    "6:Informational",
+    "7:Debug",
+    "uknown"
+};
+
+static const char *dim_facil_str[SYSLOG_FACIL_ARR_SIZE] = {
+        "0:kernel",
+        "1:user-level",
+        "2:mail",
+        "3:system",
+        "4:sec/auth",
+        "5:syslog",
+        "6:lpd/printer",                                                 
+        "7:news/nntp",     
+        "8:uucp",  
+        "9:time",
+        "10:sec/auth",
+        "11:ftp",
+        "12:ntp",
+        "13:logaudit",
+        "14:logalert",
+        "15:clock",                                                 
+        "16:local0",     
+        "17:local1", 
+        "18:local2", 
+        "19:local3", 
+        "20:local4", 
+        "21:local5", 
+        "22:local6",
+        "23:local7", 
+        "uknown"
+};
+
+void systemd_chart_init(struct File_info *p_file_info){
```

Comment:
These function initiating with `systemd` is also a concern for me, please, add a prefix that we can easily identify as our function